src/components/card/Category.jsx

```diff
@@ -23,32 +28,45 @@ const Category = () => {
         console.log("error!! ====> ", err);
       });
   }, [header]);
+  const indexOfLastPost = currentPage * itemsPerPage;
+  const indexOfFirstPost = indexOfLastPost - itemsPerPage;
 
   const getColumnsForRow = () => {
     let items = headers.map(({ id, image, name, episode, season, price }) => {
       return (
-        <Col key={id}>
-          <OneCard>
-            <CardBody>
-              <CardImg image={image} />
-              <CardTitle name={name} />
-              <CardSubtitle season={season} episode={episode} />
-              <CardSubtitle price={price} />
-            </CardBody>
-          </OneCard>
-        </Col>
+        <>
+          <Col key={id}>
+            <OneCard>
+              <CardBody>
+                <CardImg image={image} />
+                <CardTitle name={name} />
+                <CardSubtitle season={season} episode={episode} />
+                <CardSubtitle price={price} />
+              </CardBody>
+            </OneCard>
+          </Col>
+        </>
       );
     });
-    return items;
+    const currentItems = items.slice(indexOfFirstPost, indexOfLastPost);
+
+    return currentItems;
```

Comment:
You can consolidate the logic here but not defining `currentItems`, and just doing:

`return items.slice(indexOfFirstPost, indexOfLastPost)`